tests/benchmarks/cli/stories/use-cases/test_updates.py

```diff
@@ -0,0 +1,16 @@
+import os
+
+
+def test_changes(bench_dvc, tmp_dir, dvc, make_dataset):
+    dataset = make_dataset(
+        cache=False, files=True, dvcfile=False, commit=False, remote=False
+    )
+    bench_dvc("import-url", str(dataset), "new")
+
+    # Remove one file
+    for path in dataset.rglob("*"):
+        if path.is_file():
+            os.remove(path)
+            break
+    bench_dvc("update", "new.dvc")
+    bench_dvc("update", "new.dvc", name="noop")
```

Comment:
you can probably just put it into `tests/benchmarks/cli/commands/test_update.py`, as this is not really a full-blown use-case like doc use-cases.